pkg/crc/machine/machine.go

```diff
@@ -214,11 +216,32 @@ func Start(startConfig StartConfig) (StartResult, error) {
 		logging.InfoF("Starting OpenShift cluster ... [waiting 3m]")
 	}
 	result.KubeletStarted = kubeletStarted
-	//
 
 	// If no error, return usage message
 	if result.Error == "" {
 		time.Sleep(time.Minute * 3)
+		// Checks after 3 mins wait all the operators are in 'Available' state.
+		// If any of the operator still not available then wait for 2 more mins and tell users
```

Comment:
Shouldn't we be doing this for 6 minutes total, and check every minute (or even more often than that?)? If all operators are done in 1 minute, I think we can tell the user everything is fine right away rather than waiting for 2 more minutes?

Reply:
@cfergeau there is no way operator are done within the min. this check we are adding because those are not even done in 3 mins.

Reply:
It would still be more consistent to do the check every minute even if it's not going to be ready after 3 minutes. Maybe in the future they will make improvements and the cluster wil be readymuch faster :)

Reply:
but wihtin a minute you could at least check some status and possibly report progress.

Reply:
@gbraad within the minute there is no `co` resource created.

Reply:
we do not talk about created, but status changes to be 'ready/available'... right? and 'each minute' to test and report about this sounds to me a reasonable expectation. The user needs feedback... but rather have him wait for 3 minutes before anything? By that time he might conclude the process hangs ...

Reply:
@gbraad they way we are now getting the operator response back using `oc get co` and within the 1-2 mins time `co => clusteroperator` resources is not even present so the unmarshal will fail then and there. For waiting time feedback for every operation we need to introduce a timedots like we have in minishift but that would be a separate issue/PR.